e2e/src/HomeFeed.spec.js

```diff
@@ -0,0 +1,48 @@
+import { reloadReactNative } from './utils/retries'
+import { quickOnboarding, sleep, waitForElementId } from './utils/utils'
+
+const jestExpect = require('expect')
+
+beforeAll(async () => {
+  await quickOnboarding()
+})
+
+beforeEach(async () => {
+  await reloadReactNative(false)
+})
+
+// iOS only as getAttributes on multiple elements is not supported on Android
+describe('Home Feed :ios:', () => {
```

Comment:
How is this skipped on Android?

Reply:
If `:ios:` or `:android:` is in the describe block name or test name it will only attempt the suite / test on that OS.